### ISSUE TITLE
Use content sectioning

### ISSUE THREAD

**Issue by orwell1984** [phylogeny-explorer/explorer#92]
We should start implementing semantic HTML5 as a way to enable accessibility.

See great documentation at https://developer.mozilla.org/en-US/docs/Web/HTML/Element#Content_sectioning for examples.

At the very minimum, clade info should be an <article>, navigation should be marked appropriately, and we should mark footer and header.
